config-example.yaml

```
@@ -40,6 +40,11 @@ grpc_listen_addr: 127.0.0.1:50443
 # are doing.
 grpc_allow_insecure: false
 
+# Use separate a certificate for gRPC, this overwrites
+# the global certificate.
+grpc_tls_cert_path: ""
+grpc_tls_key_path: ""
```

At this point, with the growing amount of options it should end up like:
```suggestion
grpc:
  allow_insecure: false
  tls:
    cert_path: ""
    tls_key_path: ""
```

---

agreed, though this would create breaking changes, which @ohdearaugustin asked to avoid

---

It should be possible to map the allow insecure to the new value.